src/util.js

```diff
@@ -576,7 +576,7 @@ function printNumber(rawNumber) {
       // Remove unnecessary scientific notation (1e0).
       .replace(/^([\d.]+)e[+-]?0+$/, "$1")
       // Make sure numbers always start with a digit.
-      .replace(/^\./, "0.")
+      .replace(/^(\+|\-)?\./, "$10.")
```

Comment:
This is better written as `/^([+-])?\./`

Reply:
Yep I noticed that I was doing the scientific notation :)